.gitignore

```diff
@@ -3,3 +3,5 @@ bin
 obj
 *.user
 **/Properties/launchSettings.json
+
+/src/Draco.Editor.Web/app/node_modules/
```

Comment:
You could define a gitignore in the root of the project to just ignore `node_modules`, that's what the VSC extension is doing.